dashboard/app/views/levels/editors/fields/_panels.html.haml

```diff
@@ -1,7 +1,7 @@
-%h1.control-legend{data: {toggle: "collapse", target: "#panels-container"}}
+%h1.control-legend{data: {toggle: "collapse", target: "#panels-outer-container"}}
   Panels
 
-#panels-container.in.collapse
+#panels-outer-container.in.collapse
```

Comment:
There is already a `#panels-container` [here](https://github.com/code-dot-org/code-dot-org/blob/5e9c500e05ed181c1194a8f7159d96032b47d140/apps/src/panels/PanelsView.tsx#L157).